platform/os_windows.go

```diff
@@ -118,3 +120,59 @@ func KillProcessByName(processName string) {
 	cmd := fmt.Sprintf("taskkill /IM %v /F", processName)
 	ExecuteCommand(cmd)
 }
+
+// CreateExtHnsNetwork creates ext HNS network if not present.
+func CreateExtHnsNetwork(createExtNetworkType string) error {
+	if len(strings.TrimSpace(createExtNetworkType)) == 0 {
+		return nil
+	}
+
+	if !strings.EqualFold(createExtNetworkType, "l2bridge") &&
+		!strings.EqualFold(createExtNetworkType, "l2tunnel") {
+		return fmt.Errorf("Invalid ext hns network type %s", createExtNetworkType)
+	}
+
+	log.Printf("[Azure CNS] CreateExtHnsNetwork")
+	extHnsNetwork, err := hcsshim.GetHNSNetworkByName("ext")
+
```

Comment:
check the err from GetHNSNetworkByName

Reply:
We don't want to check that because first param will be nil if it cannot find the network. I'll change that param to _ .